### PR TITLE
Adds fty-info selftest

### PR DESCRIPTION
**Pull request by jana-rapava**

### CODE REVIEW

File: src/fty_info_server.c [hunk 1]
```diff
@@ -142,7 +142,11 @@ fty_info_server (zsock_t *pipe, void *args)
                  }
                  if (streq(command, "VERSION")) {
                     zmsg_t *reply = zmsg_new ();
+                    // are we on RC?
                     char *version = s_readall ("/etc/bios-release.json");
```
Comment (jimklimov) [inline src/fty_info_server.c]: Nowadays this is called `/etc/release-details.json` (`*bios*` one is a symlink that may be gone sometime later).
And containers (at least those instantiated by `ci-reset-virtual-machine.sh` should have this file too.

File: src/fty_info_server.c [hunk 2]
```diff
@@ -142,7 +142,11 @@ fty_info_server (zsock_t *pipe, void *args)
...
+                    if (version == NULL)
+                        // maybe we are on the container
+                        version = s_readall ("/usr/share/bios-web/image-version.txt");
```
Comment (jimklimov) [inline src/fty_info_server.c]: Do we (or the receipient) do anything with the version, e.g. parsing, or is it just stored and displayed? The text file snippet and the JSON one have very different contents and parsing-friendliness ;)

File: src/fty_info_server.c (hunk 1 above)
Comment (jana-rapava) [inline src/fty_info_server.c]: Thanks for letting me know, I will change the test accordingly.

File: src/fty_info_server.c (hunk 2 above)
Comment (jana-rapava) [inline src/fty_info_server.c]: This is just a proof-of-concept before I have the opportunity to discuss the interface with the Convergence team.

**Pull request merged**